docs/source/tutorials_source/devtools-integration-tutorial.py

```diff
@@ -201,6 +201,19 @@ def forward(self, x):
 # sphinx_gallery_end_ignore
 inspector.print_data_tabular()
 
+# sphinx_gallery_start_ignore
+# Display the actural inspector tabular table image
+import matplotlib.image as mpimg
+import matplotlib.pyplot as plt
+
+# Load and display the image
+img = mpimg.imread("tabular_result.png")
+fig, ax = plt.subplots(figsize=(8, 6), dpi=150)  # Adjust figsize and dpi as needed
+ax.imshow(img)
+ax.axis("off")  # Hide axes
+plt.show()
+# sphinx_gallery_end_ignore
+
 # sphinx_gallery_start_ignore
 inspector_patch.stop()
 inspector_patch_print.stop()
```

Comment:
lines 217 - 219 cause the Out: False. 
I don't know if they are needed. If you remove the the example seems to still work but the extraneous output disappears.